tests/sqllogictests/suites/ee/01_ee_system/01_0000_system_indexes

```diff
@@ -27,13 +27,13 @@ DROP TABLE IF EXISTS t1
 statement ok
 DROP TABLE IF EXISTS t2
 
-statement ok
+statement error mysql client error: Server error: `ERROR HY000 \(1105\): LicenseKeyInvalid\. Code: 1402, Text = use of aggregate_index requires an enterprise license\. current license is invalid for test_tenant, cause: LicenseKeyParseError\. Code: 1401, Text = jwt claim decode failed, cause: Token has expired\.\.'
```

Comment:
@ZhiHanZ for help

Reply:
This is because there is no license key, I will restore the reverted code in a future PR.

Reply:
This is a bug, `ee` tests should work.

Reply:
It's ok? I see you resolved the comment @BohuTANG